src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java

```diff
@@ -22,6 +26,7 @@
 public class BlockRegistry {
     public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, MineBound.MOD_ID);
 
+    public static final RegistryObject<Block> ENERGIZED_IRON_BLOCK = BLOCKS.register("energized_iron_block", () -> new Block(Properties.copy(Blocks.IRON_BLOCK).strength(3)));
```

Comment:
you can remove the strength here; we can just inherit Iron; besides, I think this is mining time, not explosion resistance (hardness)